client/src/views/Profile/Card/index.vue

```diff
@@ -21,35 +34,41 @@
 import { mapGetters, mapActions } from 'vuex'
 import CardFooter from './cardFooter'
 import FollowButton from './followBtn'
+import Avatar from '../../../components/Avatar'
 
 export default {
-  computed: {
-    ...mapGetters({ followingIdsGetter: 'profile/followingIdsGetter' }),
-    lastProfileId () {
-      return localStorage.getItem('lastProfileId')
-    },
-    currentUserId () {
-      return localStorage.getItem('profileId')
-    }
+  components: {
+    CardFooter,
+    FollowButton,
+    Avatar
+  },
+  props: {
+    profile: Object
   },
-  methods: mapActions(
-    { uploadCurrentFollowings: 'profile/uploadCurrentFollowings' }
-  ),
   data () {
     return {
-      popularAccountLimit: 100,
-      isFollowing: false
+      popularAccountLimit: 1
     }
   },
-  props: {
-    profile: Object
+  computed: {
+    ...mapGetters({ myProfile: 'profile/myProfile' }),
+    currentUserId () {
+      return localStorage.getItem('profileId')
+    },
+    popularAccount () {
+      return this.profile.followersCount > this.popularAccountLimit
+    }
   },
-  components: {
-    CardFooter,
-    FollowButton
+  async created () {
+    await this.getMyProfile()
   },
-  mounted () {
-    this.uploadCurrentFollowings(this.currentUserId)
+  methods: {
+    ...mapActions({
+      getMyProfile: 'profile/getMyProfile'
+    }),
+    updateFollowersCount (value) {
+      this.profile.followersCount = this.profile.followersCount + value
```

Comment:
this.profile.followersCount += value;

Less code and less rewriting if variable name would change.